src/pages/EPRians/EPRiansPage.style.js

```diff
@@ -21,8 +21,8 @@ S.TitleContainer = styled.div`
   align-items: center;
   justify-content: center;
 
-  width: 90rem;
-  height: 32rem;
+  width: 100vw;
+  height: 35.5vw;
```

Comment:
```suggestion
  height: 32rem;
```
높이는 그대로 고정하는 걸로 통일해요~!